googletest/include/gtest/gtest-printers.h

```diff
@@ -117,8 +117,28 @@
 #include "absl/types/variant.h"
 #endif  // GTEST_HAS_ABSL
 
+#if GTEST_HAS_STD_STRINGVIEW
+#include <string_view>
+#endif  // GTEST_HAS_STD_STRINGVIEW
+
 namespace testing {
 
+// create an alias for string_view, wherever it comes from
+#if GTEST_HAS_ABSL
+
+using absl::string_view;
```

Comment:
This is a neat trick! but is this a good idea to do this in a .h file ? hope this doesn't cause future contributors confusion or additional linking problems.

Reply:
That saves a lot of `#if's`, but I get your point. How about assigning the typedef to a dedicated name? E.g.:
`using gtest_string_view = /*...*/`

Reply:
perhaps that would be better